packages/react/ds-app-launchpad/src/ui/FileTree/Provider.stories.tsx

```diff
@@ -0,0 +1,137 @@
+/* @canonical/generator-ds 0.9.0-experimental.1 */
+
+import type { Meta, StoryFn } from "@storybook/react";
+import { useState } from "storybook/internal/preview-api";
+import { FileTree } from "./index.js";
+import type { FileTreeData, ProviderOptions } from "./types.js";
+
+const meta = {
+  title: "FileTree",
```

Comment:
It feels a bit confusing to me that this component is named in the file-structure as `FileTree` but is called `TreeView` in the stories, is there a strong reason for that?

Reply:
`TreeView` is also a thing but is a part of `FileTree`, so `FileTree.TreeView` which is a render helper for when consuming a file tree from an API.

Reply:
Makes sense, thanks